xDrip/Scenes/Entries/EditFoodEntry/Helpers/EmojiDataSource.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  EmojiDataSource.swift
+//  xDrip
+//
+//  Created by Ivan Skoryk on 05.06.2020.
+//  Copyright © 2020 Faifly. All rights reserved.
+//
+
+struct EmojiSection {
```

Comment:
Why isn't it located inside `EditFood` scene? It seems to be applicable only there.